documentation/book/proc-upgrading-brokers-newer-kafka.adoc

```diff
@@ -69,35 +78,80 @@ spec:
+kubectl logs -f _<cluster-operator-pod-name>_ | grep -E "Kafka version upgrade from [0-9.]+ to [0-9.]+, phase ([0-9]+) of \1 completed"
 ----
-Alternatively, if the old and new versions of Kafka have the same interbroker protocol version, look in the Cluster Operator logs for an `INFO` level message  in the following format:
 +
-[source,subs="+quotes"]
+[source,shell,subs=+quotes]
 ----
-Reconciliation #_<num>_(watch) Kafka(_<namespace>_/_<name>_): Kafka version upgrade from _<from-version>_ to _<to-version>_, phase 1 of 1 completed
+kubectl get po -w
 ----
 +
-For example, using `grep`:
+endif::Kubernetes[]
+On {OpenShiftName}, use:
 +
-[source,shell,subs="+quotes"]
+[source,shell,subs=+quotes]
 ----
 oc logs -f _<cluster-operator-pod-name>_ | grep -E "Kafka version upgrade from [0-9.]+ to [0-9.]+, phase ([0-9]+) of \1 completed"
 ----
++
+[source,shell,subs=+quotes]
+----
+oc get po -w
+----
++
+====
+If the current and new versions of Kafka have different interbroker protocol versions, check the Cluster Operator logs for an `INFO` level message:
+
+[source,shell,subs=+quotes]
+----
+Reconciliation #_<num>_(watch) Kafka(_<namespace>_/_<name>_): Kafka version upgrade from _<from-version>_ to _<to-version>_, phase 2 of 2 completed
+----
+Alternatively, if the current and new versions of Kafka have the same interbroker protocol version, check for:
 
-. Upgrade all your client applications to use the new version of the client libraries.
+[source,shell,subs=+quotes]
+----
+Reconciliation #_<num>_(watch) Kafka(_<namespace>_/_<name>_): Kafka version upgrade from _<from-version>_ to _<to-version>_, phase 1 of 1 completed
+----
+====
++
+The updates:
++
+* Ensure each pod is using the broker binaries for the new version
+* Configures the brokers to send messages using the interbroker protocol of the new version
 +
-WARNING: You cannot downgrade after completing this step. If, for whatever reason, you need to revert the update at this point, follow the procedure xref:proc-downgrading-brokers-older-kafka-{context}[]. 
+NOTE: Clients are still using the old version, so brokers will convert messages to the old version before sending them to the clients. To minimize this additional load, updates the clients as quickly as possible.
 
-. If the log message format versions, as identified in step 1, are the same proceed to the next step. 
+. Upgrade all client applications (consumers and producers) to use the new version of the client binaries.
++
+See xref:con-strategies-for-upgrading-clients-str[]
++
+WARNING: You cannot downgrade after completing this step. If you need to revert the update at this point, follow the procedure xref:proc-downgrading-brokers-older-kafka-{context}[].
+
++
+If required, set the version property for Kafka Connect and Mirror Maker as the new version of Kafka:
++
+.. For Kafka Connect, update `KafkaConnect.spec.version`
+.. For MIrror Maker, update `KafkaMirrorMaker.spec.version`
```

Comment:
Typo: Mirror Maker

Reply:
Good spot! I did a double take to see it.